plugins/inputs/sqlserver/sqlserver.go

```diff
@@ -260,6 +262,11 @@ func (s *SQLServer) gatherServer(server string, query Query, acc telegraf.Accumu
 	} else {
 		// when password is not provided in connection string, use AAD auth with an MSI token
 
+		// return error if AAD Auth is used for DatabaseType SQLServer since AAD Auth is only supported for Azure SQL Database or Azure SQL Managed Instance
```

Comment:
Please shorten this to `AAD Auth is only supported for Azure SQL Database or Azure SQL Managed Instance`.

Reply:
done